### PR TITLE
Make it funk

### PR DESCRIPTION
**Pull request by SimenB**
image

### CODE REVIEW

File: webpack/webpack.config.js [hunk 1]
```diff
@@ -46,19 +46,19 @@ export default {
     plugins: [
         definePlugin(),
         new webpack.DllReferencePlugin({
-            context: path.join(__dirname),
```
Comment (SimenB) [inline webpack/webpack.config.js:49]: This is the fix

File: package.json [hunk 2]
```diff
@@ -79,17 +79,17 @@
         "extract-text-webpack-plugin": "^1.0.1",
         "form-data": "^1.0.0-rc4",
         "html-loader": "0.4.3",
-        "html-webpack-plugin": "2.9.0",
```
Comment (SimenB) [inline package.json:82]: You need at least `2.10.0`

**Comment by GuillaumeCisco** [comment]
It works!
Thank you very much @SimenB 
I've tried all the context path without success and could not understand.
Is the needed version of html-webpack-plugin is written somewhere? I have totally missed it!
Just saw it in peerDependencies in your package.json.
As I remember now I've seen an Error while npm install it but thought it was not a big deal.
Hope it will help others people ;)

**Comment by SimenB** [comment]
Context should be the same as the context of normal webpack build, which usually is cwd, unless you set it to something else.
The needed version complains at install time, but I can add a note to the README.

**Comment by SimenB** [comment]
https://github.com/SimenB/add-asset-html-webpack-plugin/commit/0ccf3f517eb057f1f11394c8c9e49c28bbd35ef1

**Comment by GuillaumeCisco** [comment]
Adding it in the README will be great!

Thank you again ;)

**Comment by SimenB** [comment]
@GuillaumeCisco BTW, as of SimenB/add-asset-html-webpack-plugin#6 (current release, `0.2.0`) you can pass an array to a single instance of `AddAssetHtmlPlugin` instead of having multiple instances

**Comment by GuillaumeCisco** [comment]
Nice!
Thank you for the tip!

**Pull request merged**